AtelierTomato.Markov/AtelierTomato.Markov.Generation/MarkovChainKeyword.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AtelierTomato.Markov.Data;
+using Microsoft.Extensions.Options;
+
+namespace AtelierTomato.Markov.Generation
+{
+	public class MarkovChainKeyword(ISentenceAccess sentenceAccess, IOptions<MarkovGenerationOptions> options, Keyword keyword) : MarkovChain(sentenceAccess, options)

```

Comment:
MarkovChainKeyword reads as a Keyword that relates to MarkovChain
this seems like the opposite, a Keyword-aware Markov chain.